validation/chOpenApiValidator.go

```diff
@@ -246,3 +234,33 @@ func handleParseError(pe *openapi3filter.ParseError) models.CHError {
 func findRoute(r routers.Router, req *http.Request) (route *routers.Route, pathParams map[string]string, err error) {
 	return r.FindRoute(req)
 }
+
+func (chv *CHValidatorImpl) GetSchema(ctx context.Context, openApiSpec, contextId string) error {
```

Comment:
As per previous comment, if this does need to be exported we will need to comment it's behaviour

Reply:
Same as above